libs/src/services/solana/transfer.ts

```diff
@@ -292,14 +289,11 @@ export const createTransferInstructions = async ({
 
   const ethAddressArr = SolanaUtils.ethAddressToArray(senderEthAddress)
   const transferDataInstr = Uint8Array.of(1, ...ethAddressArr)
+  const transferInstruction = new TransactionInstruction({
+    keys: accounts,
+    programId: claimableTokenProgramKey.toString() as unknown as PublicKey,
+    data: Buffer.from(transferDataInstr)
+  })
 
-  const instructions = {
-    secpTransactionInstruction,
-    transferInstruction: new TransactionInstruction({
-      keys: accounts,
-      programId: claimableTokenProgramKey.toString() as unknown as PublicKey,
-      data: Buffer.from(transferDataInstr)
-    })
-  }
-  return instructions
+  return [secpTransactionInstruction, transferInstruction]
```

Comment:
Doublechecking that we don't call this fn anywhere else – this change is better tho, nice